### ISSUE TITLE
Term Request: Dendritic keratitis

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11761]
**Preferred term label:**
Dendritic keratitis

**Synonyms:**
* Keratitis dendritica
* Dendritic corneal ulcer
* Herpetic dendritic keratitis
* Herpes simplex dendritic keratitis
* Dendritic epithelial keratitis

**Definition (free text):**
A superficial corneal inflammation characterized by branching, linear epithelial ulcerations with terminal bulb-like swellings (dendrites) that stain with fluorescein and rose bengal. It is the hallmark epithelial manifestation of active herpes simplex virus (HSV-1) infection of the cornea and is associated with localized loss of corneal sensation.

**Parent term (use hpo.jax.org/app):**
Keratitis (HP:0000491)
*(Alternativ: Ulcerative keratitis [HP:0000529])*

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
